codestyle/cpp/CLionFormat.xml

```diff
@@ -0,0 +1,58 @@
+<code_scheme name="Project" version="173">
```

Comment:
I'm getting weird formatting for 
```c++
  switch (container->tag()) {
    case CONTAINER_TAG_STACK:
      break;
    case CONTAINER_TAG_LOCAL:
      incrementRC</* Atomic = */ false>(container);
      break;
    /* case CONTAINER_TAG_FROZEN: case CONTAINER_TAG_SHARED: */
    default:
      incrementRC</* Atomic = */ true>(container);
      break;
  }
```
->
```c++
    switch (container->tag()) {
        case CONTAINER_TAG_STACK:break;
        case CONTAINER_TAG_LOCAL:incrementRC</* Atomic = */ false>(container);
            break;
            /* case CONTAINER_TAG_FROZEN: case CONTAINER_TAG_SHARED: */
        default:incrementRC</* Atomic = */ true>(container);
            break;
    }
```

Reply:
That's "keep simple cases in one line" in "'switch' statement" option firing. Unfortunately, this particular option doesn't seem to get serialized to the xml file.